envoy/datadog_checks/envoy/data/conf.yaml.example

```diff
@@ -16,25 +16,37 @@ instances:
   #   tags:
   #     - instance:foo
 
-  #   <<<Note>>> The Envoy admin endpoint does not support auth until:
-  #   https://github.com/envoyproxy/envoy/issues/2763
-  #   For an alternative, see:
-  #   https://gist.github.com/ofek/6051508cd0dfa98fc6c13153b647c6f8
+  # You can also whitelist/blacklist metrics using regular expressions.
```

Comment:
It is not clear how this is going to work when both `metric_whitelist` and `metric_blacklist` are set when reading this doc. 
Maybe having examples could help.

Reply:
Added note in readme!